conf/steps/chrcheck.config

```diff
@@ -36,18 +36,30 @@ process {
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
             enabled: false
         ]
-        tag = {"${meta.id} ${meta.chr}"}
     }
 
     withName: 'NFCORE_PHASEIMPUTE:CHRCHECK_.*:VCF_CHR_RENAME_BCFTOOLS:BCFTOOLS_ANNOTATE' {
         ext.args   = ["-Oz", "--no-version", "--write-index=tbi"].join(' ')
-        ext.prefix = { "${meta.id}_chrrename" }
+        ext.prefix = { "${meta.id ? meta.id : meta.panel_id}_chrrename" }
         publishDir = [ enabled: false ]
     }
 
     withName: 'NFCORE_PHASEIMPUTE:CHRCHECK_.*:VCF_CHR_RENAME_BCFTOOLS:VCF_CHR_RENAME:GAWK' {
         ext.args2  = 'BEGIN {FS="\\t"} NR==1 { if ($1 ~ /^chr/) { col1=""; col2="chr" } else { col1="chr"; col2="" } } { sub(/^chr/, "", $1); if ($1 ~ /^[0-9]+|[XYMT]$/) print col1$1, col2$1; else print $1, $1 }'
-        ext.prefix = { "${meta.id}_chrrename" }
+        ext.prefix = { "${meta.id ? meta.id : meta.panel_id}_chrrename" }
+        ext.suffix = "txt"
+        publishDir = [ enabled: false ]
+    }
+
+    withName: 'NFCORE_PHASEIMPUTE:CHRCHECK_.*:VCF_CHR_RENAME_BCFTOOLS:BCFTOOLS_ANNOTATE' {
+        ext.args   = ["-Oz", "--no-version", "--write-index=tbi"].join(' ')
+        ext.prefix = { "${meta.id ? meta.id : meta.panel_id}_chrrename" }
+        publishDir = [ enabled: false ]
+    }
+
+    withName: 'NFCORE_PHASEIMPUTE:CHRCHECK_.*:VCF_CHR_RENAME_BCFTOOLS:VCF_CHR_RENAME:GAWK' {
+        ext.args2  = 'BEGIN {FS="\\t"} NR==1 { if ($1 ~ /^chr/) { col1=""; col2="chr" } else { col1="chr"; col2="" } } { sub(/^chr/, "", $1); if ($1 ~ /^[0-9]+|[XYMT]$/) print col1$1, col2$1; else print $1, $1 }'
+        ext.prefix = { "${meta.id ? meta.id : meta.panel_id}_chrrename" }
```

Comment:
You seem to have duplicated blocks for the same processes. This is defined above.